### PR TITLE
[incubator/sparkoperator] Fix a typo in Configuration section

### PR DESCRIPTION
**Pull request by mqasimsarfraz**
Fix the typo in spark operation configuration section.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@MQasimSarfraz 
Would you mind to bump the chart to latest version please? Thank you.

File: stable/spark/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: spark
-version: 1.0.0
+version: 1.0.1
```
Comment (liyinan926) [inline stable/spark/Chart.yaml]: Looks like you bumped the wrong version in the `spark` chart. You need to bump the version of the `sparkoperator` chart.

Comment (mqasimsarfraz) [reply stable/spark/Chart.yaml]: Sorry about that. I fixed that.

**Pull request merged**